schemas/Clinical/index.js

```diff
@@ -141,6 +143,25 @@ const typeDefs = gql`
     donorId: String!
   }
 
+  type ClinicalSubmissionError implements ClinicalEntityError @cost(complexity: 5) {
+    type: String!
+    message: String!
+    row: Int!
+    field: String!
+    value: String!
+    donorId: String!
+  }
+
+  type ClinicalSubmissionSchemaError implements ClinicalEntityError {
```

Comment:
If youre going to move the complexity from the interface you need to specify it here.

Reply:
just realized this right after asking for review. I've made it 10 cause there's some wrangling to flatten the upstream data :)